extensions/jenkins/src/lib/api.ts

```diff
@@ -105,9 +106,15 @@ export class JenkinsAPI {
   }
 
   async request(url: RequestInfo, init?: RequestInit) {
-    const httpsAgent = new https.Agent({
-      rejectUnauthorized: !this.jenkins.unsafeHttps,
-    });
+    let urlAgent;
```

Comment:
The variable is not assigned a value

Reply:
Oh wow, thanks for catching it 😓.
I will fix this!